status_page.html

```diff
@@ -4,6 +4,16 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
     </head>
     <style>
+        @font-face {
+            font-family: "Norse";
+            src: url("fonts/Norse.otf") format("opentype");
```

Comment:
hmmmmmmmmmmmmmmmmmmmmm alright you've convinced me that its worth having a font i have to host myself, but now the build process needs to be adjusted to push those too (basically these and the icon need to be moved to dist).  you let me know if you want to try to add that to the DeployTask file or not

Reply:
and then since we're doing that anyway sigh i guess at least an assets/ file, and maybe i want to make a be and fe subdir in here